doc/website/blog/serverless-graphql-using-aws.md

```diff
@@ -0,0 +1,556 @@
+
+### Setting up AWS Lambda
+
+With the AppSync API in place, our next stop is the AWS Lambda function to run Ent.
+For this, we navigate to the AWS Lambda service through the navbar, which leads us to the landing page of the AWS Lambda service showing our list of functions:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of AWS Lambda landing page listing functions" src="https://entgo.io/images/assets/appsync/function-list.png" />
+  <p style={{fontSize: 12}}>AWS Lambda landing page showing functions.</p>
+</div>
+We click the "Create function" button on the top right and select "Author from scratch" in the upper panel.
+Furthermore, we name the function "ent", set the runtime to "Go 1.x", and click the "Create function" button at the bottom.
+We should then find ourselves viewing the landing page of our "ent" function:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of AWS Lambda landing page listing functions" src="https://entgo.io/images/assets/appsync/function-overview.png" />
+  <p style={{fontSize: 12}}>AWS Lambda function overview of Ent function.</p>
+</div>
+Before reviewing the Go code and uploading the compiled Go code, we need to adjust some default settings of the function.
+First, we change the default handler name from `hello` to `main`, which equals the filename of the compiled Go binary:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of AWS Lambda landing page listing functions" src="https://entgo.io/images/assets/appsync/runtime-settings.png" />
+  <p style={{fontSize: 12}}>AWS Lambda runtime settings of Ent function.</p>
+</div>
+Second, we add an environment the variable `DATABASE_URL` encoding the database network parameters and credentials:
+<div style={{textAlign: 'center'}}>
+  <img alt="Screenshot of AWS Lambda landing page listing functions" src="https://entgo.io/images/assets/appsync/envars.png" />
+  <p style={{fontSize: 12}}>AWS Lambda environemnt variables settings of Ent function.</p>
+</div>
+The database URL follows the schema `postgres://username:password@hostname/dbname`.
+By default, AWS Lambda encrypts the environment variables, making them a fast and safe mechanism to supply database connection parameters.
+Alternatively, one can supply an ARN ID to an AWS Secretsmanager secret from which the function requests new credentials.
+Using AWS Secretmanager supports rotating credentials but requires an additional request at the start of each Lambda.
+A third option is to use AWS IAM to handle the database authorization.
+The former two options make only sense when using AWS RDS.
+
+If you created your Postgres database in AWS RDS, the default username and database name are `postgres`.
+The password can be reset by modifying the AWS RDS instance.
+Be aware that AWS RDS is a rather expensive service and that the network configuration is highly non-trivial!
```

Comment:
```suggestion
Be aware that AWS RDS is a rather expensive service and that the network configuration is not trivial!
```

Reply:
I removed this part because I found it too opinionated.